### PR TITLE
{lib}[foss/2016b] Tensorflow v1.2.1 with GPU support

### PR DESCRIPTION
**Pull request by schiotz**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by schiotz** [comment]
Comments:

This PR depends on PR #4871 since it is an add-on to that PR, and this specific Tensorflow variant uses the same prerequisites (included in €4871).

This PR will probably fail the self-checks even with #4871 merged, since the sanity check fails unless the machine has an NVIDIA GPU installed.

Question for @boegel: This is the first time I use a specific suffix for a special version of an easyconfig.  Please check that I did the right thing, or tell me what to change.

File: easybuild/easyconfigs/t/Tensorflow/Tensorflow-1.2.1-foss-2016b-GPU-Python-3.5.2.eb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+pyver = '3.5.2'
+pyshortver = '35'
+
+dependencies = [
+    ('Python', pyver),
+    ('protobuf-python', '3.3.0', pythonsuffix),
+    ('CUDA', '8.0.44', '', ('dummy', '')),
```
Comment (verdurin) [inline easybuild/easyconfigs/t/Tensorflow/Tensorflow-1.2.1-foss-2016b-GPU-Python-3.5.2.eb]: I think you want `True` here and in the line below - see the Travis complaint

**Comment by schiotz** [comment]
@verdurin Thanks!

I have fixed it.  I did not know one could just write True, but I can see that it is documented.  However, the docs indicate that True is just an alternative to ('dummy', ''); has it become mandatory to use it?  http://easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html#specifying-dependencies-using-dummy-toolchain

**Comment by verdurin** [comment]
@schiotz There may be a mismatch between the documentation and the tests running in Travis. Can bring this up with @boegel when he returns.

Meanwhile, there is a missing dependency:
```
Irresolvable dependencies encountered: protobuf-python/3.3.0-foss-2016b-Python-3.5.2
```

**Comment by schiotz** [comment]
@verdurin Yes, the missing dependency is in PR #4871, so this PR depends on that one.  Can I formally mark it as such?

**Comment by boegel** [comment]
@schiotz There's no way to tell Travis that this PR depends on another; once #4871 is merged, you can close/re-open this PR to rerun the tests in Travis, or push another commit (e.g. to also here use the construct where you don't run the risk out getting the Python version and `pyshortver` out of sync, cfr. https://github.com/easybuilders/easybuild-easyconfigs/pull/4871#discussion_r132718197)

**Comment by schiotz** [comment]
@boegel 
As requested, I triggered a new Travis test now that PR #4871 has been merged.  It worked successfully, so IMHO this follow-up pull request is now ready.

File: easybuild/easyconfigs/t/Tensorflow/Tensorflow-1.2.1-foss-2016b-GPU-Python-3.5.2.eb [hunk 2]
```diff
@@ -0,0 +1,64 @@
+easyblock = 'Bundle'
+
+name = 'Tensorflow'
```
Comment (boegel) [inline easybuild/easyconfigs/t/Tensorflow/Tensorflow-1.2.1-foss-2016b-GPU-Python-3.5.2.eb]: @schiotz We renamed this to `TensorFlow` (i.e. the official "spelling") in #5495, do you mind making that change here too?

Comment (schiotz) [reply easybuild/easyconfigs/t/Tensorflow/Tensorflow-1.2.1-foss-2016b-GPU-Python-3.5.2.eb]: Done!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
r10n6 - Linux red hat enterprise linux computenode 6.5, Intel(R) Xeon(R) CPU E5-2650 v3 @ 2.30GHz, Python 2.6.6
See https://gist.github.com/48dc43da81ecb67e4e96b08677d0c027 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @schiotz!

**Pull request merged**